frontend/src/pages/main/generationQR/generatedQR/QrCode.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const QrCode: React.FC = () => {
+    return (
+        <a>
+            QrCode
+        </a>
+    );
+}
+
+export default QrCode;
```

Comment:
What is this component about ?

Reply:
this page contains generated QR

Reply:
I did not see special page for the generated code. Moreover, I don't think it's a good decision. I think we should show the code on the modal verb on the same page